api/audit/models.py

```diff
@@ -117,10 +117,11 @@ def create_record(
         obj,
         obj_type,
         log_message,
-        author,
+        author=None,
```

Comment:
I don't think we need these changes to `create_record` given that you ended up changing the logic to just create the `AuditLog` directly via the constructor?

Reply:
Yep, that's correct